BackendNew/models/students.js

```diff
@@ -0,0 +1,22 @@
+// Import database connection
+const sql = require("../models/database.js");
+
+exports.getStudents = callback => {
+  var sqlString =
+    "SELECT S.NameWithInitials as Name, S.Batch as RegNo, B.Name as InternBatch FROM Student S, Batch B WHERE S.BatchId = B.BatchId";
```

Comment:
What is mean by S.Batch as RegNo?

Reply:
The Field of S>Batch should be changed as RegNo. It is the index number of the students

Reply:
Student.batch means index number of student. Is it? I got confused with the term "batch"

Reply:
Yeah I have to change the table structure. I will update it